README.md

```diff
@@ -3,13 +3,27 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  Asymptotic analysis bases an algorithms performace largly on input sizes, and how an algorithm opperates based on these input sizes. Actual runtime can be affected by factors that are not accounted for when an algorithm runs. A few of these factors include:
+  1) Asymptotic analysis looks strictly at the opperations that are being excecuted on an input, and how long they take in general. Software and hardware can be a factor that gets overlooked in this, as some machines may result in diffent complexities than others.
+  2) Asymptotic analysis looks at a sort of "whole picture" when deciding how long an algorithm takes to run, and ignores opperations that take constant time to complete. This constant time can add up and take longer than expected when working with large input sizes, resulting in a longer runtime than a $\Theta$ representation could suggest.
```

Comment:
Not sure what you mean here -- we're not ignoring operations that take constant time.